diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/BlockStructureBraces.kt

```diff
@@ -221,14 +221,14 @@ class BlockStructureBraces(private val configRules: List<RulesConfig>) : Rule("b
         allMiddleSpace: List<ASTNode>,
         node: ASTNode,
         keyword: IElementType) {
-        allMiddleSpace.forEach {
-            if (checkBraceNode(it, true)) {
+        allMiddleSpace.forEach { allMidSpace ->
```

Comment:
```suggestion
        allMiddleSpace.forEach { space ->
```
I think it's better not to have `all` in name for an individual list member.